internal/telemetry/telemetry.go

```diff
@@ -138,6 +138,15 @@ func withAuthMethod() func(Event) {
 	}
 }
 
+func withService() func(Event) {
+	return func(event Event) {
+		event.Properties["service"] = config.Service()
+		if config.OpsManagerURL() != "" {
+			event.Properties["ops_manager_url"] = config.OpsManagerURL()
```

Comment:
I'm adding ops manager url (base url) for us to be able to exclude cloud-dev from analysis